docs/getting-started.md

```diff
@@ -26,7 +26,7 @@ powerfulseal --help # or seal --help
 
 To start the web interface, use flags `--server --server-host [HOST] --server-port [PORT]` when starting PowerfulSeal in autonomous mode and visit the web server at `http://HOST:PORT/`.
 
-Python 3.6, Python 3.7 and Python 3.8 are supported.
```

Comment:
Dropping support for 3.6 may be an issue for CentOS users where 3.6 is the latest officially supported version of python. Was there a specific need to drop this support?

Reply:
To streamline and minimise the cost of ownership. If you can't get python 3.7 you can run the docker image, which is probably more convenient anyway.

We've been running python 3.7 as the base of the docker image for a while now.

Reply:
In our company, we have a standard, harden CentOS base image we are supposed to be using for all of our images, so using the pre-built seal image isn't an option.  We'll have to install 3.7 (or more likely 3.8) from source rather than a simple `yum install`.  Not a major issue, but something I suppose we'll have to deal with.